influxdb_iox/tests/end_to_end_cases/querier/influxrpc/read_filter.rs

```diff
@@ -306,6 +307,51 @@ pub async fn read_filter_periods_multi_field_predicate2() {
     .await
 }
 
+#[tokio::test]
+pub async fn read_filter_multi_data_frame() {
```

Comment:
I suggest leaving a pointer to the ticket so future readers can understand why breaking up the frames is important

```suggestion
// Test for https://github.com/influxdata/influxdb_iox/issues/7663
pub async fn read_filter_multi_data_frame() {
```